overview/default.nix

```
@@ -102,17 +102,28 @@ let
     projects.one =
       name: project:
       let
-        examples = eval {
-          imports = [ ./content-types/example-list.nix ];
-          examples = map (value: {
-            inherit (value)
-              description
-              module
-              name
-              tests
-              ;
-          }) (pick.examples project);
-        };
+        breadcrumbs = ''
+          <nav aria-label="Breadcrumb" class="breadcrumb">
+            <ol>
+              <li>
+                <a href="/">Projects</a>
+              </li>
```

```suggestion
              <li><a href="/">Projects</a></li>
```

This gets rid of the little extra whitespace after "Projects" that makes the space to the left of the slash larger than on the right.